services/rollout-service/pkg/argo/argo_test.go

```diff
@@ -367,24 +374,6 @@ func TestArgoConsume(t *testing.T) {
 						},
 					},
 				},
-				{
```

Comment:
Why did you remove this one? Shoulnd't the test continue to work just with fewer resulting calls?